css/style.css

```diff
@@ -1,7 +1,242 @@
-body{
-    margin: 0 1rem;
-}
+
+/* Extra small devices (phones, less than 768px) */
 .header{
-    background-color: #F72210;
-    color: #FFF;
-}
\ No newline at end of file
+  justify-content: start;
```

Comment:
`SUGGESTION`

Consider using a bootstrap class instead. _Note that the project is about using bootstrap._

Reply:
done